wire-library/wire-gradle-plugin/src/test/kotlin/com/squareup/wire/gradle/WirePluginTest.kt

```diff
@@ -1307,6 +1307,46 @@ class WirePluginTest {
     buildCacheDir.deleteRecursively()
   }
 
+  @Test
+  fun sourcePathOrdering() {
+    val generatedProto = "build/generated/source/wire/com/squareup/geology/Period.kt"
+
+    val fixtureRoot = File("src/test/projects/sourcepath-ordering")
+    val result = gradleRunner.runFixture(fixtureRoot) {
+      withArguments("generateProtos", "--stacktrace", "--info").build()
+    }
+
+    assertThat(result.task(":generateProtos")).isNotNull()
+    assertThat(result.output).contains("Writing com.squareup.geology.Period")
+    assertThat(result.task(":generateMainProtos")?.outcome).isEqualTo(TaskOutcome.SUCCESS)
+    assertThat(result.task(":generateProtos")?.outcome).isEqualTo(TaskOutcome.SUCCESS)
+    assertThat(File(fixtureRoot, generatedProto)).exists()
+
+    // We expect the sourceInputs property to have locations ordered consistently
```

Comment:
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

```suggestion
    // We expect the sourceInputs property to have locations ordered consistently.
```